docs/index.rst

```diff
@@ -3,22 +3,18 @@ AiiDA ABC tutorial
 
 .. _ABC_2021_Homepage:
 
-.. todo::
-
-   Update versions to ones from Quantum Mobile image
-
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 | Related resources                                                                                                           |
 +=================+===========================================================================================================+
-| python packages | `aiida-core XXXX`_, `aiida-quantumespresso XXXX`_                                                         |
+| python packages | `aiida-core 1.3.0`_, `aiida-quantumespresso 3.0.0`_                                                       |
 +-----------------+-----------------------------------------------------------------------------------------------------------+
-| codes           | `Quantum ESPRESSO XXXX`_                                                                                  |
+| codes           | `Quantum ESPRESSO 6.5`_                                                                                   |
```

Comment:
Same here, unfortunately. I left these as XXXX's to fix when the AiiDAlab cluster was up and running. On https://aiidalab-demo.materialscloud.org/ the Quantum ESPRESSO version is still 6.0.